bpf/lib/common.h

```diff
@@ -325,7 +325,9 @@ struct tunnel_value {
 	__u16 pad;
 } __packed;
 
-#define ENDPOINT_F_HOST		1 /* Special endpoint representing local host */
+#define ENDPOINT_F_HOST			1 /* Special endpoint representing local host */
+#define ENDPOINT_F_ATHOSTNS		2 /* Endpoint located at the host networking namespace */
+#define ENDPOINT_MASK_HOST_DELIVERY	(ENDPOINT_F_HOST | ENDPOINT_F_ATHOSTNS)
```

Comment:
Nice, I like the mask approach for compatibility!

Future task will be to switch over the existing `cilium_host` endpoints to set `ENDPOINT_F_ATHOSTNS`, but not `ENDPOINT_F_HOST`. And then remove the special handling in `icmp6.h`.